ODIN_II/SRC/simulate_blif.cpp

```diff
@@ -3907,17 +3917,12 @@ static void write_vector_to_file(lines_t *l, FILE *file, int cycle)
 
 		if (line_has_unknown_pin(line, cycle) || num_pins == 1)
 		{
-			if ((num_pins + 1) > BUFFER_MAX_SIZE)
```

Comment:
why did you delete these error messages??

Reply:
No more buffer to overflow, stale check

Reply:
also if undefined value are x by default